### PR TITLE
Add aquainfra processes

### PR DESCRIPTION
**Pull request by MarkusKonk**

### CODE REVIEW

File: ecology.yaml [hunk 1]
```diff
@@ -895,6 +895,10 @@ tools:
     owner: ecology
     tool_panel_section_label: 'Get Data'
 
+  - name: aquainfra
```
Comment (bgruening) [inline ecology.yaml:898]: Oo, was that really the tool ID? This is way to generic :(

Comment (MarkusKonk) [reply ecology.yaml:898]: Yeah, I also thought that this morning, but I thought it's too late to change that...

Comment (MarkusKonk) [reply ecology.yaml:898]: @bgruening  Shall I just update the tool with a more specific id?

Comment (bgruening) [reply ecology.yaml:898]: until now its not installled please change that ...

**Comment by bgruening** [comment]
Wann brauchst du es?

**Comment by MarkusKonk** [comment]
Ich brauche das nicht zwingend diese Woche. Wenn das Samstag online geht, reicht das wohl.

**Pull request merged**